rubocop/cop/i18n/static_translation_definition.rb

```diff
@@ -0,0 +1,116 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module I18n
+      # This cop flags translation definitions in static scopes because changing
+      # locales has no effect and won't translate this text again. The rule here is based upon
+      # https://gitlab.com/gitlab-org/gitlab/-/blob/master/rubocop/cop/static_translation_definition.rb?ref_type=heads.
+      #
+      # @example
+      #   # bad
+      #   class MyExample
+      #     # Constant
+      #     Translation = I18n.t('A translation.')
+      #
+      #     # Class scope
+      #     field :foo, title: I18n.t('A title')
+      #
+      #     validates :title, :presence, message: I18n.t('is missing')
+      #
+      #     # Memoized
+      #     def self.translations
+      #       @cached ||= { text: I18n.t('A translation.') }
+      #     end
+      #
+      #     included do # or prepended or class_methods
+      #       self.error_message = I18n.t('Something went wrong.')
+      #     end
+      #   end
+      #
+      #   # good
+      #   class MyExample
+      #     # Keep translations dynamic.
+      #     Translation = -> { I18n.t('A translation.') }
+      #     # OR
+      #     def translation
+      #       I18n.t('A translation.')
+      #     end
+      #
+      #     field :foo, title: -> { I18n.t('A title') }
+      #
+      #     validates :title, :presence, message: -> { I18n.t('is missing') }
```

Comment:
Validations can take a symbol for the message which will be looked up relative to the model in the same way as the [default error messages](https://guides.rubyonrails.org/i18n.html#error-message-scopes). I cannot find docs for this right now, but it is used by the [blocking code](https://github.com/otwcode/otwarchive/blob/acebcaddee50ce6d6d1503f22666ea47b6891f91/app/models/comment.rb#L54), [en.yml](https://github.com/otwcode/otwarchive/blob/acebcaddee50ce6d6d1503f22666ea47b6891f91/config/locales/models/en.yml#L114).

```suggestion
      #     validates :title, :presence, message: :very_custom_missing
```